supervisord/datadog_checks/supervisord/supervisord.py

```diff
@@ -182,3 +204,13 @@ def _build_message(proc):
 Exit Status: %(exitstatus)s"""
             % proc
         )
+
+    def _collect_metadata(self, supe):
+        try:
+            version = supe.getSupervisorVersion()
+        except Exception as e:
+            self.log.warning("Error collecting version: %s", e)
+            return
+
+        self.log.debug('Version collected: %s', version)
+        self.set_metadata('version', version, scheme='regex', pattern=SUPERVISORD_VERSION_PATTERN)
```

Comment:
Should `final_scheme` be semver?

Reply:
Shouldn't `version.scheme` correspond to `version.raw` ? or something else ?

Currently, the `version.scheme` is using the check name in this case `'version.scheme': 'supervisord'`.